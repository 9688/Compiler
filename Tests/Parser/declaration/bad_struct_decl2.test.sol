<--------TABLES-------->
<---TABLE--->
VARS::

TYPES::
char char

double double

int int

void void

Ошибка синтаксиса в строке 1 позиции 9: отсутствует спецификатор типа

