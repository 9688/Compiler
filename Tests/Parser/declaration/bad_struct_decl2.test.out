<<<table #1>>>
<tags>
<types>
int
char
double
void
<vars>
Семантическая ошибка в строке 1, позиции 9: необходим хотя бы один элемент

